bindist/linux/debian/scripts/build.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+import os
+import sys
+import json
+import subprocess
+import shutil
+import datetime
+
+os.environ["DEBIAN_FRONTEND"] = "noninteractive"
+
+DISTRIBUTION=sys.argv[1]
+NOW=datetime.datetime.utcnow().strftime("%Y%m%dT%H%M%SZ")
+
+SCRIPT_DIR=os.path.dirname(os.path.abspath(__file__))
+BUILDINFO_DIR=os.path.abspath(os.path.join(SCRIPT_DIR, "..", DISTRIBUTION))
+BUILD_DIR=os.path.abspath(os.path.join(BUILDINFO_DIR, "build"))
+
+class cd:
+    """
+    Context manager for changing the current working directory
+
+    Credits: https://stackoverflow.com/a/13197763
+    """
+    def __init__(self, new_path):
+        self.new_path = os.path.expanduser(new_path)
+
+    def __enter__(self):
+        self.old_path = os.getcwd()
+        os.chdir(self.new_path)
+
+    def __exit__(self, etype, value, traceback):
+        os.chdir(self.old_path)
+
+def run(*args, **kwargs):
+  """Same as subprocess.run(), but forces 'check' to True"""
+  kwargs["check"] = True
+  print("+", " ".join(args[0]))
+  return subprocess.run(*args, **kwargs)
+
+def get_buildinfo():
+  """"Parse and return 'buildinfo.json'"""
+  path = os.path.join(SCRIPT_DIR, "..", DISTRIBUTION, "buildinfo.json")
+  return json.load(open(path, "rb"))
+
+def apt_install(pkgs):
+  """Install a package using system's 'apt'"""
+  return run("apt install {} -y".format(" ".join(pkgs)).split())
```

Comment:
We should use `apt-get` instead of `apt` for scripts, `apt` isn't necessarily stable